Basic-Car-Maintenance/Documentation.docc/branch-updates.tutorial

```diff
@@ -0,0 +1,62 @@
+@Tutorial(time: 10) {
+    @Intro(title: "Updating Feature Branch") {
+        During the PR process, it is important to make sure the local feature branch has been updated with the latest `dev` branch changes before pushing updates to the PR. Follow these instructions to keep a feature branch up to date with the latest `dev` changes during the PR process.
+
+    }
+    
+    @Section(title: "Grab latest dev Changes") {
+        @ContentAndMedia {
+            Update local your dev branch with changes from the remote dev branch.
+        }
+        
+        @Steps {
+            @Step {
+                Sync your forked repo with the latest changes from Dev branch by clicking the `Sync fork` button and following the prompts.
+                
+                @Image(source: 9-sync.png, alt: "GitHub repository sync button" )
+            }
+            @Step {
+                In a command prompt, use `git checkout dev` to checkout the dev branch in your local repo.
+                @Image(source: 10-checkout-dev.png, alt: "Command prompt using git checkout dev")
+            }
+            
+            @Step {
+                Update the remote tracking branches by using `git fetch`. This will 
+                Note - `git fetch` is optional as it is done within a `git pull`. For a more in-depth discussion on these commands, see [Git Pull GitHub Guide](https://github.com/git-guides/git-pull)
+                
+                @Image(source: 11-fetch.png, alt: "Command prompt using git fetch")
+            }
```

Comment:
```suggestion
            @Step {
                Update the remote tracking branches by using `git fetch`.
                
                Note - `git fetch` is optional as it is done within a `git pull`. For a more in-depth discussion on these commands, see [Git Pull GitHub Guide](https://github.com/git-guides/git-pull)
                
                @Image(source: 11-fetch.png, alt: "Command prompt using git fetch")
            }
```

This reformats it to show the note separated, more like a note, as opposed to together with the step itself


<img width="352" alt="Screenshot 2023-10-29 at 1 39 15 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/18070f8f-c6ff-4cb8-92ed-0111896a962d">